orc8r/cloud/go/services/orc8r_worker/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package orc8r_worker
+
+const (
+	ServiceName = "orc8r_worker"
```

Comment:
Nit: use a const.go for this instead of doc.go (ik we do it both ways)

Reply:
I can change the other ones into const.go -- just wondering, but is there a reason we prefer one over the other (besides for consistency reasons?)

Reply:
Reason is doc.go is expected to only contain comments (documentation), is idiosyncratic/confusing when it contains real code

Also yes that would be great, but I already have it marked as a bootcamp task 🙂